calculator/src/Components/Keypad.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+class Keypad extends React.Component {
+  render() {
+    const {
+      btnPressed,
+      calculate,
+      clear,
+      handleDecimal,
+      toggleSign,
+      addOperand,
+      subtractOperand,
+      divisionOperand,
+      multiplicationOperand,
+    } = this.props;
+    return (
+      <section className="button">
+        <button onClick={btnPressed} value="7">
+          7
+        </button>
+        <button onClick={btnPressed} value="8">
+          8
+        </button>
+        <button onClick={btnPressed} value="9">
+          9
+        </button>
+        <button onClick={multiplicationOperand} value="*">
+          x
+        </button>
+        <button onClick={divisionOperand} value="/">
+          ÷
+        </button>
+        <button onClick={btnPressed} value="6">
```

Comment:
An alternative would be to handle the values in the onClick. 
onClick={() => handleClick("6") }

Could you make each button a reusable component?